shortcut/linux.py

```diff
@@ -1,36 +1,52 @@
 import os
 import stat
-from .exception import *
+# from .exception import *
 from .base import ShortCutter
-            
+
+
 class ShortCutterLinux(ShortCutter):
-    
     def _get_desktop_folder(self):
-        return os.path.join(os.path.join(os.path.expanduser('~')), 'Desktop')
-        
+        import subprocess
+        return subprocess.check_output([
+            'xdg-user-dir',
```

Comment:
I dont know a great deal about the deployment of xdg-user-dir, is it on most linux distros? I checked on ubuntu and raspbian and it seems to be available,

Reply:
I wasn't able to google another way to find out where is the desktop in case of a non-english Ubuntu installation. I guess might be better to try-catch and use 'desktop' on fail.